src/components/unstable/Select/utils.ts

```diff
@@ -30,7 +30,7 @@ export const getFlattenOptions = (options: (SelectOption | SelectOptgroup)[]): F
     }, [] as FlattenOption[]);
 };
 
-export const getListboxItemHeight = (args: {
+export const getSelectPopupItemHeight = (args: {
```

Comment:
"Select" part can be omitted like in the function below `getPopupVerticalOffset `